google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -1440,6 +1440,15 @@ def compose sources, destination, acl: nil, encryption_key: nil
         #   using the URL, but only when the file resource is missing the
         #   corresponding values. (These values can be permanently set using
         #   {File#content_disposition=} and {File#content_type=}.)
+        # @param [String] scheme The URL scheme. The default value is `HTTPS`.
+        # @param [Boolean] virtual_hosted_style Whether to use a virtual hosted-style
+        #   hostname, which adds the bucket into the host portion of the URI rather
+        #   than the path, e.g. `https://mybucket.storage.googleapis.com/...`.
+        #   For V4 signing, this also sets the `host` header in the canonicalized
+        #   extension headers to the virtual hosted-style host, unless that header is
+        #   supplied via the `headers` param. The default value is `false`.
```

Comment:
Default value of 'false' uses the form of `https://storage.googleapis.com/mybucket`

Reply:
Added.